modules/develop/pages/produce-data/leader-pinning.adoc

```diff
@@ -0,0 +1,51 @@
+= Leader Pinning
+:page-categories: Clients, Development
+// tag::single-source[]
+:description: Learn about leader pinning and how to configure a preferred partition leader location based on cloud availability zones or regions.
+
+Produce requests to write data to Redpanda topics go through the topic partition leader, which syncs messages across its follower replicas. For a Redpanda cluster deployed across multiple availability zones (AZs), leader pinning ensures that a topic's partition leaders are geographically closer to clients. Leader pinning helps decrease networking costs and guarantee lower latency.
```

Comment:
```suggestion
Produce requests to write data to Redpanda topics go through the topic partition leader, which syncs messages across its follower replicas. For a Redpanda cluster deployed across multiple availability zones (AZs), leader pinning ensures that a topic's partition leaders are geographically closer to clients. Leader pinning helps decrease networking costs and guarantees lower latency.
```

Reply:
@asimms41 Meant for "helps" to apply to both "decrease" and "guarantee" - does that sound all right?

Reply:
+1 on guarantees